From 4b5249d8078b61b5c38c2432533c8df341b9417e Mon Sep 17 00:00:00 2001
From: Cheick TRAORE <cheick.traore-ext@st.com>
Date: Fri, 24 May 2024 10:37:18 +0200
Subject: [PATCH] dt-bindings: mfd: stm32-lptimer: add support for stm32mp21

Add a new stm32mp21 compatible to stm32-lptimer dt-bindings in order to
support the STM32MP21 SoC.

Change-Id: Ia948caf52db9e9d8c9a1486658857903e87f5855
Signed-off-by: Cheick TRAORE <cheick.traore-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/382337
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 Documentation/devicetree/bindings/mfd/st,stm32-lptimer.yaml | 5 +++++
 1 file changed, 5 insertions(+)

--- a/Documentation/devicetree/bindings/mfd/st,stm32-lptimer.yaml
+++ b/Documentation/devicetree/bindings/mfd/st,stm32-lptimer.yaml
@@ -23,6 +23,7 @@ properties:
   compatible:
     enum:
       - st,stm32-lptimer
+      - st,stm32mp21-lptimer
       - st,stm32mp25-lptimer
 
   reg:
@@ -61,6 +62,7 @@ properties:
       compatible:
         enum:
           - st,stm32-pwm-lp
+          - st,stm32mp21-pwm-lp
           - st,stm32mp25-pwm-lp
 
       "#pwm-cells":
@@ -78,6 +80,7 @@ properties:
       compatible:
         enum:
           - st,stm32-lptimer-counter
+          - st,stm32mp21-lptimer-counter
           - st,stm32mp25-lptimer-counter
 
     required:
@@ -91,6 +94,7 @@ properties:
       compatible:
         enum:
           - st,stm32-lptimer-timer
+          - st,stm32mp21-lptimer-timer
           - st,stm32mp25-lptimer-timer
 
     required:
@@ -105,6 +109,7 @@ patternProperties:
       compatible:
         enum:
           - st,stm32-lptimer-trigger
+          - st,stm32mp21-lptimer-trigger
           - st,stm32mp25-lptimer-trigger
 
       reg:
